queue_services/entity-emailer/src/entity_emailer/email_templates/REG-COMPLETED.html

```diff
@@ -29,31 +29,18 @@
             [[20px.html]]
             <p>The Statement of Registration can be used to open a business account with your bank/credit union.
             It is an important business form and should be stored in a safe space.
-            
+
             [[20px.html]]
             To make changes to your business or to get a copy of this Statement of Registration,
             log in to your <a href="{{ entity_dashboard_url }}">BC Business Registry account</a>.</p>
-            
+
             [[20px.html]]
             <span class="bold">Important:</span> Read the <a href="https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/business-management/small-business/small_business_resources.pdf">
               Small Business Resources guide</a> for key resources to support your small business, including how to
-              register with federal, provincial and local partners (e.g. a Business BCeID, GST, PST, WorkSafeBC, 
-              business license etc.).
-
-            [[20px.html]]
-            In addition, the following links can assist you:
-            <ul>
-              <li>
-                A message from the <a href="https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/business-management/permits-licences-and-registration/registries-other-assets/small_business_congradulations.pdf">
-                  Minister of Citizens’ Services and the Minister of Jobs, Economic Recovery and Innovation
-                </a>
-              </li>
-              <li>
-                All businesses in British Columbia, including sole proprietorships and general partnerships, must 
-                follow the rules around handling personal information set out in BC’s Personal Information Protection Act. 
-                For more information and resources, visit <a href="https://www.oipc.bc.ca/privacyright">www.oipc.bc.ca/privacyright</a>
-              </li>
-            </ul>
+              register with federal, provincial and local partners (e.g. a Business BCeID, GST, PST, WorkSafeBC,
+              business license etc.). All businesses in British Columbia, including sole proprietorships and general partnerships, must follow the
```

Comment:
Can you confirm with Andy if this sentence "All businesses in British Columbia...." should be in a separate paragraph as it is shown in updated design?